src/log.js

```diff
@@ -792,6 +824,11 @@ class InterfaceBase {
  * @implements LoggingInterface
  */
 class SimpleInterface extends /* private */ InterfaceBase {
+  constructor({ fields = {}, ...opts } = {}) {
```

Comment:
I do not think specifying fields here again will be necessary; the default in InterfaceBase should take care of that!

Reply:
good catch. probably a left over...